File: papi/lista1/exercicio13.py
nome = input("Digite o seu nome: ")
while True:
    try:
        idade = int(input("Digite a sua idade: "))
        break
    except ValueError:
        print("Idade inválida, por favor digite um número.")

print("O seu nome é", nome)
if idade >= 18:
    print("Você é maior de idade.")
else:
    print("Você é menor de idade")

File: papi/lista1/exercicio13.kt
package lista1

fun main() {
    print("Por favor informe o seu nome: ")
    var nome = readLine()!!
    var idade: Double

    while (true) {
        try {
            print("Por favor informe sua idade: ")
            idade = readLine()!!.toDouble()
            if (idade <= 0) {
                throw NumberFormatException()
            }
            break
        } catch (e: NumberFormatException) {
            print("Idade inválida. Por favor, informe um número maior que zero, com um ponto como separador decimal.")
        }
    }
    
    if (idade >= 18) {
        println("Você é maior de idade $nome!")
    } else {
        println("Você é menor de idade $nome!")
    }
}